crates/agglayer-prover-types/src/generated/agglayer.prover.v1.rs

```diff
@@ -320,8 +376,8 @@ pub mod proof_generation_service_server {
         }
     }
     /// Generated gRPC service name
-    pub const SERVICE_NAME: &str = "agglayer.prover.v1.ProofGenerationService";
-    impl<T> tonic::server::NamedService for ProofGenerationServiceServer<T> {
+    pub const SERVICE_NAME: &str = "agglayer.prover.v1.PessimisticProofService";
```

Comment:
Nit - should this be `kebab-case`?

Reply:
This is generated code from `proto` files, and the [guideline](https://protobuf.dev/programming-guides/style/#services) defines that service names should be PascalCase :)